lib/src/message/message_item_bloc.dart

```diff
@@ -205,7 +211,7 @@ class MessageItemBloc extends Bloc<MessageItemEvent, MessageItemState> {
     if (!_listenersRegistered) {
       _repositoryStreamHandler = RepositoryMultiEventStreamHandler(
         Type.publish,
-        [Event.msgDelivered, Event.msgRead],
+        [Event.msgDelivered, Event.msgRead, Event.msgFailed, Event.error],
```

Comment:
Is listening for Event.error necessary here?

Reply:
(I didn't found any place where we react to error events, therefore the question)

Reply:
That was for testing. I forgot to remove it.